draft-irtf-cfrg-opaque.md

```diff
@@ -1877,6 +1895,14 @@ implementation considerations.
   of the derivation of OPRF keys via a single PRF. As long as the derivation
   of different OPRF keys from a single OPRF has different PRF inputs, the
   protocol is secure. The choice of such inputs is up to the application.
+- {{JKX18}} comments on a strong defense against essentially the only
+  inherent remaining weakness of a "strong aPAKE", namely, offline
+  dictionary attacks upon server compromise. The authors suggest implementing
```

Comment:
not only on server compromise, but also the server itself it is more than honest-but-curious.

Reply:
Thanks!